src/components/Button.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Button({children, type, className, onClick}) {
+  return (
+    <button
+      type={type}
+      className={className}
+      onClick={onClick}
+    >{children}</button>
+  );
+}
```

Comment:
Dit is nu meer een doorgeefluik. Ten opzichte van het html button element zelf gebeurd er verder niets in dit component. Je had dus net zo goed direct kunnen gebruiken